packages/razzle/config/createConfig.js

```diff
@@ -54,6 +54,8 @@ module.exports = (
     babelrc: true,
     cacheDirectory: true,
     presets: [],
+    target,
+    env,
```

Comment:
We could move this as an additional argument to modifyBabelOptions ?

modifyBabelOptions(mainBabelOptions, { target, env })

Reply:
I moved them to a `buildOptions`.